### ISSUE TITLE
Incompatible with Expo 38

### ISSUE THREAD

**Issue by kmanj-pixel** [gorhom/react-native-bottom-sheet#19]
# Bug

The current Expo version (38) only supports react-native-reanimated 1.9. It looks this package is using useValue which was introduced in [reanimated 1.10](https://github.com/software-mansion/react-native-reanimated/releases/tag/1.10.0).

I'm not sure if there are other methods that this package depends that makes it incompatible with Expo 38.
 
https://github.com/gorhom/react-native-bottom-sheet/blob/65af6f1e8a20ba8c07e2e4192027819bb46f6f96/src/hooks/useScrollable.ts#L8

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
| @gorhom/bottom-sheet         | 1.1.1  |
| react-native                    | 0.62 (Expo 38)  |
| react-native-reanimated         | ~1.9   | 
| react-native-gesture-handler    | ~1.6.0  |

## Steps To Reproduce

Render the component and get the following error:

```
(0, _reactNativeReanimated.useValue) is not a function. (In '(0, _reactNativeReanimated.useValue)(0)', '(0, _reactNativeReanimated.useValue)' is undefined)
```

**Comment by sa8ab** [comment]
I currently use it with reanimated 2 and it is working totally fine, maybe try installing reanimated yourself again using expo/npm/yarn

**Comment by gorhom** [comment]
thanks @kmanj-pixel for reporting this issue, @sa8ab i think the problem is Expo come with reanimated embedded - not an expert/user of expo.

however the solution would be to replace `useValue` source from `react-native-reanimated` to `react-native-redash`

**Comment by wobsoriano** [comment]
So there is no way to use it with expo now?

**Comment by fr3fou** [comment]
I get the same error in v40.0.0, how can I fix it? I copied the example from the docs

**Comment by gorhom** [comment]
@wobsoriano , @fr3fou i am looking into the issue now

**Comment by gorhom** [comment]
it is working with Expo 40 @fr3fou , @wobsoriano 
https://expo.io/@gorhom/projects/bottom-sheet-v2

**Comment by fr3fou** [comment]
https://snack.expo.io/@fr3fou/81abc7 I can't get the basic example to work from the docs

**Comment by fr3fou** [comment]
![image](https://user-images.githubusercontent.com/1344906/103179523-ac1f6e00-4895-11eb-961a-15503c0cb5a0.png)

**Comment by fr3fou** [comment]
https://gorhom.github.io/react-native-bottom-sheet/usage from here

**Comment by lyubchev** [comment]
i am experiencing the same issue

**Comment by fr3fou** [comment]
Was this fixed in 2.0.2?
